test/verify/run_verify.cpp

```diff
@@ -88,7 +88,8 @@ inline void compile_check(migraphx::program& p,
     auto num = shapes.size();
     for(std::size_t i = 0; i < num; ++i)
     {
-        if(p.get_output_shapes()[i].lens() != shapes[i].lens())
+        // Dimensions check only or strides also matter?
```

Comment:
I don't understand this comment. You changed it such that it compares shape classes. So it would also check for strides.

Reply:
It's unclear to me if we wanted to only check the dimensions. I changed it to check shape to avoid calling `shape.lens()`.

Reply:
Okay. I think it should be okay to check for shape equality instead of just lens.